packages/terra-clinical-onset-picker/src/terra-dev-site/test/clinical-onset-picker/LessWeek.test.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import moment from 'moment';
+import OnsetPicker from '../../../OnsetPicker';
+
+class HandledOnsetExample extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      precision: 'before',
+      granularity: 'year',
+    };
+
+    this.handleOnset = this.handleOnset.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleOnset(onsetObject) {
+    this.setState({
+      precision: onsetObject.precision,
+      granularity: onsetObject.granularity,
+      onsetDate: onsetObject.onsetDate,
+      ageUnit: onsetObject.ageUnit,
+    });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+
+    this.setState(prevState => ({
+      submittedData: {
+        precision: prevState.precision,
+        granularity: prevState.granularity,
+        onsetDate: prevState.onsetDate,
+        ageUnit: prevState.ageUnit,
+      },
+    }));
+  }
+
+  render() {
+    const birthdate = moment().subtract(6, 'days');
+    return (
+      <div>
+        <p>Onset picker with age to be less than a week.</p>
+        <p>{`Current date locked to ${moment().format('MM-DD-YYYY')}`}</p>
```

Comment:
To make the purpose of this test more clear I feel adding birthdate also here would be appropriate.

Reply:
Fixed here https://github.com/cerner/terra-clinical/pull/543/commits/dea7f66925ff7547256295d24794eb437c9dc8ca.